Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -37,31 +37,31 @@ public struct TestCommand: AsyncParsableCommand, HasTrackableParameters {
 
     @Option(
         name: .shortAndLong,
-        help: "Test on a specific device."
+        help: "[Deprecated] Test on a specific device."
     )
     var device: String?
 
     @Option(
         name: .long,
-        help: "Test on a specific platform."
+        help: "[Deprecated] Test on a specific platform."
     )
     var platform: String?
 
     @Option(
         name: .shortAndLong,
-        help: "Test with a specific version of the OS."
+        help: "[Deprecated] Test with a specific version of the OS."
     )
     var os: String?
 
     @Flag(
         name: .long,
-        help: "When passed, append arch=x86_64 to the 'destination' to run simulator in a Rosetta mode."
+        help: "[Deprecated] When passed, append arch=x86_64 to the 'destination' to run simulator in a Rosetta mode."
     )
     var rosetta: Bool = false
 
     @Option(
         name: [.long, .customShort("C")],
-        help: "The configuration to be used when testing the scheme."
+        help: "[Deprecated] The configuration to be used when testing the scheme."
```

Comment:
Same as [here](https://github.com/tuist/tuist/pull/6300/files#diff-42704af5d5879e9018d76de29bf550baeec7fc9091f81f349c0c39abf9de6cb0R76), I wouldn't drop this one.